modules/ossm-vs-istio.adoc

```diff
@@ -89,3 +89,8 @@ spec:
 = The Istio Container Network Interface (CNI) plug-in
 
 {ProductName} includes CNI plug-in, which provides you with an alternate way to configure application pod networking. The CNI plug-in replaces the `init-container` network configuration eliminating the need to grant service accounts and projects access to Security Context Constraints (SCCs) with elevated privileges.
+
+= Envoy, Secret Discovery Service, and Certificates
+
+* The QUIC listener for Envoy is not available in {ProductName}. 
+* Secret Discovery Service is not supported because it uses hostPath mounts. 
```

Comment:
maybe `Deployment of TLS certificates using the Secret Discovery Service (SDS) functionality of Istio is not currently supported in {ProductName}, as the Istio implementation depends on a `nodeagent` container that uses hostPath mounts. `